src/Orleans.Core/Serialization/IKeyedSerializer.cs

```diff
@@ -9,5 +9,10 @@ internal interface IKeyedSerializer : IExternalSerializer
         /// Gets the identifier for this serializer.
         /// </summary>
         KeyedSerializerId SerializerId { get; }
+
+        /// <summary>
+        /// Returns <see langword="true"/> if is serializer should only be used for fallback scenarios, <see langword="false"/> otherwise.
```

Comment:
```suggestion
        /// Returns <see langword="true"/> if this serializer should only be used for fallback scenarios, <see langword="false"/> otherwise.
```